src/utils/date.js

```diff
@@ -3,12 +3,29 @@ import format from 'date-fns/format';
 import isToday from 'date-fns/is_today';
 import isYesterday from 'date-fns/is_yesterday';
 import isSameYear from 'date-fns/is_same_year';
+import addMinutes from 'date-fns/add_minutes';
 import tz from 'timezone/loaded';
 
 export { default as isSameDay } from 'date-fns/is_same_day';
 
+/**
+ * Use consistent timezone (UTC) in local and CI tests, for snapshot tests.
```

Comment:
I think this isn't quite accurate (because unfortunately the thing this function is doing / the thing the underlying APIs leave as the best we were able to find how to do is not as clean as this description). I'll push to the branch a commit which can go at the end, expanding this jsdoc.

Reply:
Ah, yeah, thanks for doing that. It's tricky! 🤕